lib/auth/supabase/middleware.ts

```
@@ -0,0 +1,80 @@
+
+  // IMPORTANT: DO NOT REMOVE auth.getUser()
+
+  const {
+    data: { user },
+  } = await supabase.auth.getUser()
+
+  // Extraire la locale de l'URL (fr, en, etc.)
+  const locale = request.nextUrl.pathname.split('/')[1] || 'en';
+  
+  // Extraire le chemin sans la locale
+  const pathWithoutLocale = '/' + request.nextUrl.pathname.split('/').slice(2).join('/');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)